From 25bc52a31577a22e82166598bb9160aad952e49a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 Nov 2019 08:55:40 +0200
Subject: [PATCH] [SW-0] rootfs: cpld: fix the bus number for stop

The CPLD is behind the multi master and hence got different bus number.
Fix the CPLD bus number in stop() routine.

Change-Id: If513522e1e435525ea9b1ad9a8c7895f45d818ef
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat b/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
index df8d695..043aa42 100755
--- a/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
+++ b/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
@@ -37,7 +37,7 @@
 		echo 0x25 > /sys/bus/i2c/devices/i2c-1/delete_device
 	;;
 	*)
-		echo 0x60 > /sys/bus/i2c/devices/i2c-1/delete_device
+		echo 0x60 > /sys/bus/i2c/devices/i2c-3/delete_device
 	;;
 	esac
 }
